src/client/utils/validations.js

```diff
@@ -180,14 +212,18 @@ function isValidVeteranAddress(data) {
       isValidField(isValidPhone, data.mobilePhone);
 }
 
+function isValidFinancialDisclosure(data) {
+  return isNotBlank(data.provideFinancialInfo.value);
+}
+
 function isValidSpouseInformation(data) {
   return (isBlankFullNameField(data.spouseFullName) || isValidFullNameField(data.spouseFullName)) &&
       isValidField(isValidSSN, data.spouseSocialSecurityNumber) &&
       (isBlankDateField(data.spouseDateOfBirth) || isValidDateField(data.spouseDateOfBirth)) &&
       (isBlankDateField(data.dateOfMarriage) || isValidDateField(data.dateOfMarriage)) &&
       (isBlankAddressField(data.spouseAddress) || isValidAddressField(data.spouseAddress)) &&
       isValidField(isValidPhone, data.spousePhone) &&
-      isNotBlank(data.maritalStatus);
+      isNotBlank(data.maritalStatus.value);
 }
 
 function isValidChildInformationField(child) {
```

Comment:
Right now the form will still validate even if `DateChildBecameDependent` is before `ChildDateOfBirth`. Maybe this should be equal to or less than?